### ISSUE TITLE
Ability to use multiple tokens for the same vault

### ISSUE THREAD

**Issue by lvets** [starkandwayne/safe#114]
Hi,

As a vault admin, I would like the ability to target the same vault (but different aliases maybe?) with different tokens.

For instance, `safe target vault1-token1` would alias to vault1 with token1. `safe target vault1-token2` on the other hand would be an alias for the same vault, but with a different token.

**Comment by jhunt** [comment]
You can alias your safe targets already.

For example, this should just work:

```
safe target http://localhost:8200 vault1
safe target http://localhost:8200 vault1-again
safe target http://localhost:8200 vault1-yet-again
```

Those all point to the same backend vault, (http://localhost:8200) but they each maintain a separate entry in `~/.saferc`, so you can use different tokens and even different authentication methods.

For a practical example, whenever I set up a Vault, I usually enable some sort of authentication like Github or LDAP.  To keep from having to re-supply my root token, I just alias `my-vault` for GH/LDAP logins and `my-vault-admin` for the root token avenue.

**Comment by lvets** [comment]
@jhunt That's not true, each entry will _not_ have their own token.

As I understand it, your example leads to the following .saferc file:
```
Aliases:
  vault1: http://localhost:8200
  vault1-again: http://localhost:8200
  vault1-yet-again: http://localhost:8200
Current: vault1-yet-again
SkipVerify:
  http://localhost:8200: false
Targets:
  http://localhost:8200: <LAST USED TOKEN>
```

**Comment by jhunt** [comment]
Huh.  Apparently, in my saferc, I have a version with the trailing / and without.

Might be time to finally re-do the saferc format to be less ... parallel.

**Comment by lvets** [comment]
Cool. Thanks!

**Comment by jhunt** [comment]
This should be fixed as of Safe 0.5.0, https://github.com/starkandwayne/safe/releases/tag/v0.5.0
